lib/cmd/fh3/appforms/data-sources/create.js

```diff
@@ -0,0 +1,21 @@
+var common = require('../../../../common.js');
+
+module.exports = {
+  'desc': 'Create A New Data Source.',
+  'examples': [{cmd: 'fhc appforms data-sources create --name="New Data Source" --description="A Data Source" --refreshInterval=2 --serviceGuid="glaovni73qwjla5qt2oyocw3" --endpoint="/path/to/data"', desc: 'Read A Single Data Source'}],
+  'demand': ['name', 'serviceGuid', 'endpoint', 'refreshInterval', 'description'],
+  'alias': {},
+  'describe': {
+    'name': "Name Of The Data Source To Create",
```

Comment:
tiny thing, but I think for consistency we probably don't want to go all caps on all words (a quick scan through other command descriptions/names etc didn't notice this elsewhere)